JavaScript FFI test fails with `Error: Cannot find module 'bn.js'`

When running the `presumbit.sh` script, the test `js_test` fails with error:

```log
10/11 Test #10: js_test ..........................***Failed    0.35 sec

> starkware_crypto_ffi@1.0.0 test /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js
> mocha


Error: Cannot find module 'bn.js'
Require stack:
- /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js/crypto.js
- /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js/test/crypto_test.js
- /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js/node_modules/mocha/lib/esm-utils.js
- /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js/node_modules/mocha/lib/mocha.js
- /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js/node_modules/mocha/lib/cli/one-and-dones.js
- /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js/node_modules/mocha/lib/cli/options.js
- /home/dev/repos/starkware/crypto-cpp/src/starkware/crypto/ffi/js/node_modules/mocha/bin/mocha
```

--- a/src/starkware/crypto/ffi/js/package.json
+++ b/src/starkware/crypto/ffi/js/package.json
@@ -15,7 +15,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "bigint-buffer": "^1.1.5",
-    "bn": "^1.0.5",
+    "bn.js": "^5.2.0",
     "ffi-napi": "^3.1.0"
   },
   "devDependencies": {

